library/src/main/java/org/jellyfin/apiclient/serialization/GsonJsonSerializer.java

```diff
@@ -1,14 +1,14 @@
 package org.jellyfin.apiclient.serialization;
 
-import org.jellyfin.apiclient.model.apiclient.ServerInfo;
-
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
+import org.jellyfin.apiclient.model.apiclient.ServerInfo;
+
 import java.io.InputStream;
 
 public class GsonJsonSerializer implements IJsonSerializer {
-    private final Gson gson;
+    public final Gson gson;
```

Comment:
Ewww public property! :nauseated_face:

Reply:
It looks ugly right now but the end goal with this change it to use Gson directly and deprecate the `IJsonSerializer` interface. This way we can test everything with a single json serializer (gson) to make sure no issues occur.
Because I use Gson directly in some places right now I made it a public property